kubectl-rook-ceph.sh

```diff
@@ -360,4 +348,17 @@ if [[ "${#REMAINING_ARGS[@]}" -eq 0 ]]; then
   fail_error "No command to run"
 fi
 
+# Default values
+NAMESPACE=$(printenv ROOK_CLUSTER_NAMESPACE ) # namespace of the cluster
```

Comment:
Lines 352-353 need to be before line 345, otherwise the env vars would overwrite the parse_flags, right?